aws_lambda_powertools/utilities/parameters/exceptions.py

```diff
@@ -9,3 +9,11 @@ class GetParameterError(Exception):
 
 class TransformParameterError(Exception):
     """When a provider fails to transform a parameter value"""
+
+
+class SetParameterError(Exception):
+    """When a provider raises an exception on setting a SSM parameter"""
+
+
+class SetSecretError(Exception):
+    """When a provider raises an exception on setting an AWS Secret"""
```

Comment:
To keep it consistent with the Get exceptions on the same file, I would recommend:

```suggestion
    """When a provider raises an exception on writing a secret"""
```